src/gameplay/game.ts

```diff
@@ -1367,6 +1376,8 @@ class Game extends Room {
       const indSpyChange =
         Math.round((teamSpyChange / this.spyUsernames.length) * 10) / 10;
 
+      // // TODO Disable the ratings 
```

Comment:
?

Reply:
Currently, Elo is being calculated here for ranked games. I'll have to disable it, so that we can calculate Glicko-2 at the end of the rating period instead.